### DISCUSSION TITLE
Resource requirements

### DISCUSSION THREAD

**Discussion by Faeez-Ali** [akvorado/akvorado#544] [Q&A]
I have configured Akvorado on my machine ( no docker) , and I am getting around 20K flows/s  , how much resources(hard-disk space) should I require. to retain the data for 1 year.  currently I am using default configuration with codec(LZ4) , but my Akvorado is occupying more space then expected ,  what step should I take to reduce the space occupied by Akvorado more specifically Clickhouse.

![akvorado](https://user-images.githubusercontent.com/64612272/223034209-14737ed1-9d77-4256-a6c1-9a2a2da725dc.PNG)

**Comment by vincentbernat** [comment]
A first step would be to change the TTL settings of the flows table. It defaults to 15 days. You can lower it to 7 days. Most of the space is on this table.
